Source/WebKit/Shared/Extensions/WebExtensionEventListenerType.h

```diff
@@ -76,6 +76,8 @@ enum class WebExtensionEventListenerType : uint8_t {
     TabsOnReplaced,
     TabsOnUpdated,
     TestOnMessage,
+    TestOnTestStarted,
+    TestOnTestFinished,
```

Comment:
You need to bump `currentBackgroundContentListenerStateVersion` when adding to this.